Libplanet.Extensions.Cocona/Commands/Key/DerivationCommand.cs

```diff
@@ -0,0 +1,34 @@
+namespace Libplanet.Extensions.Cocona.Commands.Key
+{
+    using System;
+    using global::Cocona;
+    using Libplanet.Crypto;
+
+    public class DerivationCommand
+    {
+        public void PrivateKey(
+            [Argument("PRIVATE-KEY")]
+            string privateKeyHex,
+            [Option]
+            bool publicKey = false,
+            [Option]
+            bool address = false)
+        {
+            if (!(publicKey ^ address))
+            {
+                throw new CommandExitedException($"Only one flag should be used between {nameof(publicKey)} and {nameof(address)}", -1);
+            }
```

Comment:
For better error messages, how about separating two cases — both are turned off and both are turned on?

```suggestion
            if (publicKey && address)
            {
                throw new CommandExitedException(
                   "--public-key and --address are mutually exclusive; turn on one at a time.",
                   -1
                );
            }
            else if (!publicKey && !address)
            {
                 throw new CommandExitedException(
                    "One of --public-key and --address should be turned on.",
                    -1
                 );
            }
```